examples/echo/responder.cpp

```diff
@@ -0,0 +1,18 @@
+#include "src/Responder.h"
+
+int main() {
+    Responder r = Responder();
+
+    r.on("echo", [] (const std::stringstream& input, std::stringstream& output) {
+        zclock_sleep(100);
+        output.write(input.str().data(), input.str().size());
```

Comment:
output << input.str(); ?

Reply:
I got used to working with char* in a different part of the code and accidentally used this approach in this example. Using a stream operator is much nicer.